src/activities/CreateAreaMeasurement2D.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import type {
+    IActivityHandler,
+    IActivityContext,
+} from "@geocortex/workflow/runtime/IActivityHandler";
+import { MapProvider } from "@geocortex/workflow/runtime/activities/arcgis/MapProvider";
+import { activate } from "@geocortex/workflow/runtime/Hooks";
+import AreaMeasurement2D from "@arcgis/core/widgets/AreaMeasurement2D";
+import MapView from "@arcgis/core/views/MapView";
+import { removeFunction, linearMeasurementUnits, AreaMeasurement2DResult } from "../types";
+export interface CreateAreaMeasurement2DInputs {
+    /**
+     * @description Unit system (imperial or metric) or specific unit used for area values.
+     */
+    areaUnit?: linearMeasurementUnits | string;
+
+}
+
+export interface CreateAreaMeasurementOutputs {
+    measurement: AreaMeasurement2DResult | undefined;
```

Comment:
Use `?` on all of these rather than `| undefined`. Technically they are equivalent, but the designer handles the former better.

Reply:
done